lib/rubocop/cop/rails/presence.rb

```diff
@@ -76,6 +76,8 @@ def on_if(node)
             receiver, other = redundant_negative_receiver_and_other(node)
           end
           return unless receiver
+          return if other.multiline? || other.if_type? || other.rescue_type?
```

Comment:
> `other.rescue_type?`

I'm not sure why you add the check, that is for `rescue` type. Can you add a test case(s) for the check? I can't find any test cases.